xbot_service_interface/examples/lcd_hello.py

```
@@ -0,0 +1,159 @@
+
+
+    xbot.start()
+    print("Waiting for RemoteGPIOService (id=10)…")
+
+    try:
+        counter = 0
+        while xbot.ok():
+            if svc.connected:
+                try:
+                    lcd_write(svc, f"Counter: {counter}", line=0)
+                except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)